backend/installGlobalHook.js

```diff
@@ -57,6 +65,33 @@ function installGlobalHook(window: Object) {
           this._listeners[evt].map(fn => fn(data));
         }
       },
+      // Fiber-only:
+      supportsFiber: true,
```

Comment:
If we release a new version of the devtools that auto-updates everyone then before we release Fiber we can revert this check with the assumption that devtools gets updated before renderers?